docs/api.md

```diff
@@ -1559,6 +1578,10 @@ Shortcut for [page.mainFrame().executionContext().queryObjects(prototypeHandle)]
     - `networkidle2` - consider navigation to be finished when there are no more than 2 network connections for at least `500` ms.
 - returns: <[Promise]<[Response]>> Promise which resolves to the main resource response. In case of multiple redirects, the navigation will resolve with the response of the last redirect.
 
+#### page.screencastFrameAck(sessionId)
+- `sessionId` <[number]> The ID of the frame that is being acknowledged. Provided in the `screencastframe` event.
```

Comment:
```suggestion
- `sessionId` <[number]> The ID of the frame that is being acknowledged. Provided in the `'screencastframe'` event.
```

Reply:
I don't think we need extra '' here, e.g.

"Sends a `keydown`, `keypress`/`input`, and `keyup` event for each character in the text."